pyfr/backends/cuda/blasext.py

```diff
@@ -22,7 +22,8 @@ def axnpby(self, *arr, subdims=None):
 
         # Build the kernel
         kern = self._build_kernel('axnpby', src,
-                                  [np.int32]*3 + [np.intp]*nv + [dtype]*nv)
+                                  [np.int32]*3 + [np.intp]*nv + [dtype]*nv
+                                 )
```

Comment:
Can move ) back up.

Reply:
Just following up here, and should we be setting the l1 pref? (Don't think it should make any difference, but would be interested to see.)